pages/dkp/konvoy/2.2/architecture/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: layout.pug
+navigationTitle: Architecture
```

Comment:
What are we trying to describe with this doc? Upstream Kubernetes docs already have https://kubernetes.io/docs/concepts/overview/components/, don't we want to highlight the architecture of Konvoy/DKP here instead?

Reply:
I agree... the diagram just looked kind of lonely on the page.  Should I bring the information from Choose Infrastructure to this page instead?  Or to avoid duplicate data, just delete all the extraneous blabber and just have the diagram located after the Intro to DKP page?